src/dataServices/dynamoDbUtil.ts

```diff
@@ -29,10 +30,19 @@ export class DynamoDbUtil {
         return cleanedItem;

```

Comment:
I think we're going to need to update this method to remove `ttlInSeconds` from the DDB record. `ttlInSeconds` is system generated meta data similar to `LOCK_END_TS_FIELD` and should not be returned to the user.

While we're here, should we make a TTL_IN_SECONDS_FIELD constant so we can reuse that value across classes?

Reply:
Thanks for the constant update and interface version update. 

In terms of using the `ttlInSeconds` value, does the routing package need to use this value? I'm thinking this value can be handled completely by the `persistence` package. We can add `ttlInSeconds` as an optional parameter that is passed to the constructor of `DynamoDbDataService`. If `ttlInSeconds` exist then we append that value to Create and Update requests. If not then we keep the original behavior of the class.

Reply:
Sure, the persistence constructors could take in the Map<resourceType:string, ttlInSeconds> and move the compute logic to persistence. We'll need to rollback the interface changes, add logic to pass the map to persistence constructor, move the computation, remove the passing of ttlInSeconds to the persistence methods and remove the logic for removing ttlInSeconds from API responses.

I don't feel strongly here either way, more concerned with what is best long term for code quality. If it makes sense to move all the archive logic to persistence to help encapsulate ownership and keep the layers to a minimum than I'll get started on the updates.

Reply:
@nguyen102 Updated interface, persistence, routing and deployment to have the persistence layer handle ttls. I have all of the repos up on my AWS Account but ran out of time to start testing today. I will try to have testing and any bug fixes ready by Monday morning for this PR.

Reply:
Finished up manual testing against our AWS Account's FHIR Works instance. The ttl change to being managed in persistence looks fine.

Reply:
@zambonilli Just to double check, what interface change was required to make to have persistence handle ttl? I was under the impression that no change was needed, but would like to double check with you in case I missed something.